### ISSUE TITLE
How to use Adam optimizer in Rosetta?

### ISSUE THREAD

**Issue by luo-dahui** [LatticeX-Foundation/Rosetta#98]

**Comment by yuucyf** [comment]
For example, in file:https://github.com/LatticeX-Foundation/Rosetta/blob/master/example/tutorials/code/rtt-logistic_regression_stat.py

you can replace
`tf.train.GradientDescentOptimizer(learning_rate).minimize(loss)`
with
`tf.train.AdamOptimizer(learning_rate=learning_rate).minimize(loss)`
